lib/internal/util/inspect.js

```diff
@@ -1247,27 +1266,29 @@ function improveStack(stack, constructor, name, tag) {
   let len = name.length;
 
   if (constructor === null ||
-      (name.endsWith('Error') &&
-      stack.startsWith(name) &&
+      (StringPrototypeEndsWith(name, 'Error') &&
+      StringPrototypeStartsWith(stack, name) &&
       (stack.length === len || stack[len] === ':' || stack[len] === '\n'))) {
     let fallback = 'Error';
     if (constructor === null) {
-      const start = stack.match(/^([A-Z][a-z_ A-Z0-9[\]()-]+)(?::|\n {4}at)/) ||
-        stack.match(/^([a-z_A-Z0-9-]*Error)$/);
+      // eslint-disable-next-line node-core/avoid-prototype-pollution
+      const start = RegExpPrototypeSymbolMatch(/^([A-Z][a-z_ A-Z0-9[\]()-]+)(?::|\n {4}at)/, stack) ||
+        // eslint-disable-next-line node-core/avoid-prototype-pollution
+        RegExpPrototypeSymbolMatch(/^([a-z_A-Z0-9-]*Error)$/, stack);
```

Comment:
Why not use `RegExpPrototypeExec` instead, as the linter suggests?

```suggestion
      const start = RegExpPrototypeSymbolExec(/^([A-Z][a-z_ A-Z0-9[\]()-]+)(?::|\n {4}at)/, stack) ||
        RegExpPrototypeSymbolExec(/^([a-z_A-Z0-9-]*Error)$/, stack);
```

Reply:
oh, duh, sorry. i was just disabling the linter since using RegExpPrototypeSymbolMatch was better than `.match` :-p will fix